pkg/ccr/handle/drop_rollup.go

```diff
@@ -0,0 +1,25 @@
+package handle
+
+import (
+	"github.com/selectdb/ccr_syncer/pkg/ccr"
+	"github.com/selectdb/ccr_syncer/pkg/ccr/record"
+	festruct "github.com/selectdb/ccr_syncer/pkg/rpc/kitex_gen/frontendservice"
+)
+
+func init() {
+	ccr.RegisterJobHandle[*record.DropRollup](festruct.TBinlogType_DROP_ROLLUP, &DropRollupHandle{})
+}
+
+type DropRollupHandle struct {
+	// The drop rollup binlog is idempotent
+	IdempotentJobHandle[*record.DropRollup]
```

Comment:
The drop rollup binlog is not a idempotent binlog, See determineBinlogState for details.